MANIFEST.in

```diff
@@ -6,3 +6,5 @@ prune .github
 include src/borg/platform/darwin.c src/borg/platform/freebsd.c src/borg/platform/linux.c src/borg/platform/posix.c
 include src/borg/platform/syncfilerange.c
 include src/borg/platform/windows.c
+include src/borg/testsuite/attic.tar.gz
+include src/borg/paperkey.html
```

Comment:
Maybe add a comment above these 2 lines about why they are needed (see comment in first lines)?

Reply:
done thanks